openedx/core/djangoapps/bookmarks/apps.py

```diff
@@ -0,0 +1,16 @@
+"""
+Configuration for bookmarks Django app
+"""
+from django.apps import AppConfig
+
+
+class BookmarksConfig(AppConfig):
+    """
+    Configuration class for bookmarks Django app
+    """
+    name = 'openedx.core.djangoapps.bookmarks'
+    verbose_name = "Bookmarks"
+
+    def ready(self):
+        # Register the signals handled by bookmarks.
+        from . import signals
```

Comment:
This doesn't have the unused-import warning suppression, but pylint apparently doesn't care.  Maybe because it isn't module-level?  I don't really care either way.